unit_tests/test_table.py

```diff
@@ -1441,6 +1475,11 @@ def read_rows(self, *args, **kwargs):
             curr_row_data = rows_dict.pop(row_key)
             yield curr_row_data
 
+    def sample_row_keys(self):
+        for row_key in self.initial_split_keys:
+            self.sample_row_keys_result.append(_MockSampleRowKey(row_key))
+        return self.sample_row_keys_result
```

Comment:
It would be clearer just to have the testcase initialize `sample_row_keys_result` and just return it (rather than computing it), which would also let you test for the case that the back-end returns an empty sequence (the trailing `b""` isn't guaranteed in the proto).

Reply:
added new test case https://github.com/googleapis/google-cloud-python-happybase/pull/70/commits/8220b6bf6454c43e5822743a89d10d0980505248#diff-ac84df270fae3be33d72fa312714fa13R164